packages/hint-create-element-svg/tests/tests.ts

```diff
@@ -0,0 +1,41 @@
+import generateHTMLPage from 'hint/dist/src/lib/utils/misc/generate-html-page';
+import { getHintPath } from 'hint/dist/src/lib/utils/hint-helpers';
+import { HintTest } from '@hint/utils-tests-helpers/dist/src/hint-test-type';
+import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
+
+const hintPath = getHintPath(__filename);
+
+const generateHTMLPageWithDivTag = (head: string, body: string) => {
+    return generateHTMLPage(head, `<div id="container">${body}</div>`);
+};
+const generateScriptTag = (script: string) => {
+    return `<script type="text/javascript">${script}</script>`;
+};
+
+const htmlElementCreate = 'document.getElementById("container").appendChild(document.createElement("a"));';
+const invalidCircleCreate = 'document.getElementById("container").appendChild(document.createElement("circle"));';
+const invalidSvgCreate = 'document.getElementById("container").appendChild(document.createElement("svg"));';
+const validSvgCreate = 'document.getElementById("container").appendChild(document.createElementNS("http://www.w3.org/2000/svg", "svg"));';
+
+const tests: HintTest[] = [
+    {
+        name: 'Should not use createElement to create SVG element',
+        reports: [{ message: 'Avoid using createElement to create SVG elements; use createElementNS instead' }],
```

Comment:
Let's validate we're getting the report generated at the right position in the file too.